src/components/Header/Header.tsx

```diff
@@ -1,15 +1,38 @@
+import { useLogin } from '../../routes/Login/Login.context';
 import * as Styled from './Header.styles';
+import { useNavigate } from 'react-router-dom';
 
 const Header = () => {
+    const { token, setToken } = useLogin();
+    const navigate = useNavigate();
+
+    const handleLogo = () => {
+        navigate('/');
+    };
+
+    const handleLogIn = () => {
+        navigate('/login');
+    };
+
+    const handlelogOut = () => {
```

Comment:
handlelogOut = handleLogOut ??? looks amazing tho !

Reply:
It's updated now! :)